libs/pdi-scanner/src/rust/protocol/parsers.rs

```diff
@@ -1366,7 +1366,7 @@ mod tests {
     fn test_packet_with_crc() {
         let input = b"\x02D\x03\xb4";
         let (remainder, (tag,)) = packet_with_crc((tag(b"D"),))(input).unwrap();
-        assert_eq!(remainder, []);
+        assert_eq!(remainder, b"");
```

Comment:
This started failing due to having multiple implementations that could apply to the other version. This uses a more specific type to avoid the problem.